source/code-generation.txt

```diff
@@ -0,0 +1,56 @@
+.. _rm-code-generation:
+
+===============
+Code Generation
+===============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Relational Migrator can help you convert your SQL and application code 
```

Comment:
This intro topic needs to more clearly explain the scenarios for each of the two major feature areas under code gen.

"Generate app code" is about automatically generating brand new code based on the MongoDB schema. There is no conversion involved. Users will need to figure out how to incorporate the code into their app, although the generated code is very generic.

"Query converter" is about taking existing code - specifically stored procs, SQL queries or view definitions, and update them to work with MongoDB while taking the schema transformations into account. 

The intro section and the use cases need to make this clear.

Reply:
I updated the table with a bit of extra content to drive home these points.